.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Might be a good idea to add this to the `.gitignore`, that way git won't accidently pick up config files.